config/transformer/models_list

```diff
@@ -4,5 +4,11 @@ openconfig-acl.yang
 openconfig-acl-annot.yang
 openconfig-network-instance.yang
 openconfig-network-instance-annot.yang
+sonic-vxlan.yang
```

Comment:
Please remove sonic*.yang. The models_list file is intended only for openconfig yang. The sonic yangs with annotation files are automatically handled by transformer.